network-api/networkapi/donate/pagemodels/help_page.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.fields import StreamField
+from wagtail.models import Page
+from wagtail_localize.fields import SynchronizedField, TranslatableField
+
+from networkapi.donate.models import BaseDonationPage
+from networkapi.wagtailpages.pagemodels.customblocks.base_fields import base_fields
+
+
+class DonateHelpPage(BaseDonationPage):
+    template = "donate/pages/help_page.html"
+
+    parent_page_types = ["DonateLandingPage"]
+
+    subpage_types: list = []
+
+    max_count = 1
+
+    show_notice = models.BooleanField(default=False, help_text="Show delayed response notice")
+
+    body = StreamField(base_fields, blank=True, use_json_field=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel("show_notice"),
+        FieldPanel("body"),
+    ]
+
+    translatable_fields = [
+        # Promote tab fields
+        SynchronizedField("slug"),
+        TranslatableField("seo_title"),
+        SynchronizedField("show_in_menus"),
+        TranslatableField("search_description"),
+        SynchronizedField("search_image"),
+        # Content tab fields
+        SynchronizedField("show_notice"),
+        SynchronizedField("body"),
```

Comment:
Again, this will probably be handled in the other PR, but let's confirm the type of translation here

Reply:
Thanks Jhonatan, this was updated to `TranslatableField` in PR #11182